resources/tests/factories.py

```diff
@@ -62,7 +63,7 @@ class Meta:
 
 
 class AbstractAreaSectionFactory(factory.django.DjangoModelFactory):
-    identifier = factory.Faker("random_letter")
+    identifier = factory.fuzzy.FuzzyText(length=30)
```

Comment:
The fuzzy stuff [is deprecated](https://factoryboy.readthedocs.io/en/stable/fuzzy.html). What's the issue that this was causing?

If we need a unique value, should use [sequences](https://factoryboy.readthedocs.io/en/stable/reference.html?highlight=sequence#sequence)

Reply:
Ah, so it seems. I'll swap it to a sequence.

The issue was [this error](https://github.com/City-of-Helsinki/berth-reservations/pull/519/checks?check_run_id=2295013788#step:6:1575) in another PR. There were two piers with the same identifier in `payments/schema/mutations.py:799` which caused it to choose the wrong one and finding no berths in `payments/schema/mutations.py:808`.